openapi-docs/git_push.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
@vickywane this has to be tied to semantic release.

Reply:
This script is not being used. The generated files are tracked by git in the parent folder.

I would add this script as part of the files to be deleted after each generation since it's not being used.